deployment/1_setup_test_environment.js

```diff
@@ -27,22 +28,23 @@ async function deployENSRegistry(deployer, owner, domain) {
   const domainName = parts[0];
 
   // Create the 'eth' and 'xyz' namespaces
-  const setSubnodeOwnerXYZ = await ENSWrapper.contract.setSubnodeOwner(BYTES32_NULL, utils.sha3(extension), owner);
+  const setSubnodeOwnerXYZ = await ENSWrapper.contract.setSubnodeOwner(BYTES32_NULL, utils.sha3(extension), owner, { gasPrice });
   await ENSWrapper.verboseWaitForTransaction(setSubnodeOwnerXYZ, `Setting Subnode Owner for ${extension}`);
 
   // Create the 'argentx.xyz' wallet ENS namespace
-  const setSubnodeOwnerArgent = await ENSWrapper.contract.setSubnodeOwner(utils.namehash(extension), utils.sha3(domainName), owner);
+  const setSubnodeOwnerArgent = await ENSWrapper.contract.setSubnodeOwner(utils.namehash(extension), utils.sha3(domainName), owner, { gasPrice });
   await ENSWrapper.verboseWaitForTransaction(setSubnodeOwnerArgent, `Setting Subnode Owner for ${domainName}.${extension}`);
 
   return ENSWrapper.contractAddress;
 }
 
 // For development purpose
 async function deployKyber(deployer) {
+  const { gasPrice } = deployer.defaultOverrides;
   const KyberWrapper = await deployer.deploy(Kyber);
   const ERC20Wrapper = await deployer.deploy(ERC20, {}, [KyberWrapper.contractAddress], TEST_ERC20_SUPPLY, TEST_ERC20_DECIMALS);
 
-  const addToken = await KyberWrapper.contract.addToken(ERC20Wrapper.contractAddress, TEST_ERC20_RATE, TEST_ERC20_DECIMALS);
+  const addToken = await KyberWrapper.contract.addToken(ERC20Wrapper.contractAddress, TEST_ERC20_RATE, TEST_ERC20_DECIMALS, { gasPrice });
```

Comment:
`Initializing UniswapFactory` step is also missing `gasPrice` here.

Reply:
Done.